pkg/core/object/target.go

```diff
@@ -1,4 +1,4 @@
-package transformer
+package object
```

Comment:
correct, but how about locate package in the `pkg/services/object/internal` if it's not intended to be used outside Object service?